examples/nsl_pk/DY.Example.NSL.Protocol.Stateful.Proof.fst

```diff
@@ -143,6 +141,7 @@ let protocol_invariants_nsl_has_nsl_event_invariant = all_events_has_all_events
 
 (*** Proofs ***)
 
+#push-options "--fuel 1"
```

Comment:
As you noticed, this is related to the definition of `rand_just_generated` using the new `is_not_empty`.
I will close the discussion here, since we can discuss that further at the def of `rand_just_generated`.